src/HealthChecks.Publisher.Seq/SeqOptions.cs

```diff
@@ -6,7 +6,7 @@ public class SeqOptions
     {
         public string Endpoint { get; set; } = null!;
 
-        public string ApiKey { get; set; } = null!;
+        public string? ApiKey { get; set; }
```

Comment:
I had a look at the official API client for seq and it seems that the `ApiKey` can be null so it probably makes sense for it to be null here too?

https://github.com/datalust/seq-api/blob/dev/src/Seq.Api/Client/SeqApiClient.cs#L75=

Reply:
yes